.github/ISSUE_TEMPLATE/suggest-new-kit.yaml

```diff
@@ -0,0 +1,40 @@
+name: KIT-Proposal
+description: Suggest a new KIT
+title: "[NEW KIT]: "
+labels: ["kit-proposal"]
+assignees:
+  - octocat
+body:
+  - type: input
+    id: KIT-contact
+    attributes:
+      label: Contact email
```

Comment:
Yes that is be a possibility. I had a meeting with @danielmiehle and @maximilianong about the content of the Issue and they preferred to have the e-mail address present.

Reply:
I think for now we would need it. I would like to stick to it.

Reply:
@maximilianong ... sure we need an email information but this referenced or linked to the GitHub user preferred notifications and at least you can find the email information on each user profile. In my opinion it is doubled.
little example for my public profile
![image](https://user-images.githubusercontent.com/121097161/236841303-a54f211f-34a7-4c58-b3c0-8f77998f2f8e.png)

Reply:
Ok - that's fine. Maybe I skipped this while reading it ^^

Reply:
pushed the new changes that removed the contact email field

Reply:
thx to @LuisRickert 😉 

question for me is now, why every repository within Eclipse Tractus-X should have this issue templates? Should / Could every repository have the option to generate a kit?

If not, for me it is better located under repository 
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io

Reply:
Valid. Added it to this repo, because I found the other template here. Will move the templates to [this PR](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/212).

Reply:
could also do it in 2 PR´s, makes it easier to verify your changes